src/components/Avatar/Avatar.tsx

```diff
@@ -23,6 +23,71 @@ export type IAvatar<T = {}> = AvatarProps &
 
 const cnAvatar = cn('Avatar');
 
+const colors = [
+  '#F9A825',
+  '#F56B00',
+  '#C62828',
+  '#C2185B',
+  '#E92064',
+  '#911FA2',
+  '#9010D1',
+  '#7448DD',
+  '#3E3ACB',
+  '#2044D5',
+  '#016BDC',
+  '#2196F3',
+  '#00BCD4',
+  '#0088A3',
+  '#008F7C',
+  '#00A352',
+  '#1ABC1A',
+  '#6FB80A',
+];
+
+const getRandom = (min: number, max: number) => {
+  return Math.floor(Math.random() * (max + 1 - min)) + min;
+};
+
+const getBackgroundColor = (name: string | undefined) => {
```

Comment:
я бы как-то отразила в названии, что для одного имени всегда вернется один цвет, иначе можно подумать что он совсем рандомно может выбираться

```suggestion
const getBackgroundColorForName = (name: string | undefined) => {
```

Reply:
Переименовал на `getColorForName`